config_templates/gretel/synthetics/high-accuracy.yml

```diff
@@ -11,12 +12,17 @@
 # at the expense of increased training and generation time.
 
 schema_version: "1.0"
-
+name: "high-accuracy"
 models:
   - synthetics:
       data_source: "__tmp__"
       params:
         epochs: 100
         vocab_size: 0
         learning_rate: .001
-        rnn_units: 256
\ No newline at end of file
+        rnn_units: 256
+        reset_states: false
+        # Gretel privacy filtering
+        privacy_filters:
```

Comment:
@yamini this might be one level in too far

Reply:
Thanks for catching that.